robot/EDA/resources/EDA.py

```diff
@@ -457,3 +511,6 @@ def shift_to_default_content(self):
         )
         return
 
+    def write_to_console(self, message):
```

Comment:
There is already a method, log_to_console(). Can we use that wherever we want to log the message to console? Just trying to understand the reason behind this function.

Reply:
I use this one in EDA